models/public/midasnet/model.yml

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  MidasNet is a model for monocular depth estimation trained by mixing several datasets;
+  as described in the following paper: 
+  "Towards Robust Monocular Depth Estimation: Mixing Datasets for Zero-Shot Cross-Dataset Transfer"
+  <https://arxiv.org/pdf/1907.01341>
+
+  The model input is a blob that consists of a single image of "1x3x384x384" in RGB order.
+
+  The model output is an inverse depth map that is defined up to an unknown scale factor.
+task_type: monocular_depth_estimation
+files:
+  - name: midas_net.py
+    size: 2662
+    sha256: 0bf2e5295b69b3af9adafaf97e154186d29a808325435440aa68c7aa4d25f753
+    source: https://raw.githubusercontent.com/intel-isl/MiDaS/master/models/midas_net.py
```

Comment:
Please use a specific commit hash rather than `master`. (Otherwise, there's no guarantee that the version we want will be fetched.)

Reply:
done.